fetch.bs

```diff
@@ -324,6 +393,155 @@ specialized multimap. An ordered list of key-value pairs with potentially duplic
  <li><p>Return the <a for="header">combined value</a> with <var>name</var> and <var>list</var>.
 </ol>
 
+<p>To
+<dfn export for="header list" lt="get, decode, and split|getting, decoding, and splitting" id=concept-header-list-get-decode-split>get, decode, and split</dfn>
+a <a for=header>name</a> <var>name</var> from <a for=/>header list</a> <var>list</var>, run these
+steps:
+
+<ol>
+ <li><p>Let <var>initialValue</var> be the result of <a for="header list">getting</a>
+ <var>name</var> from <var>list</var>.
+
+ <li><p>If <var>initialValue</var> is null, then return null.
+
+ <li><p>Let <var>input</var> be the result of <a>isomorphic decoding</a> <var>initialValue</var>.
+
+ <li><p>Let <var>position</var> be a <a for=string>position variable</a> for <var>input</var>,
+ initially pointing at the start of <var>input</var>.
+
+ <li><p>Let <var>values</var> be a <a for=/>list</a> of <a for=/>strings</a>, initially empty.
+
+ <li><p>Let <var>value</var> be null.
```

Comment:
Do we get anything by starting with value as null instead of the empty string?  Seems like if we had it as the empty string, it saves a line of "code".  We do end up with values being an empty string at the end of the code, but I don't think that hurts?  The code is correct, either way, just think that less complexity is better, though don't have strong feelings here.